### PR TITLE
Add DevOps Enterprise Summit

### PR DESCRIPTION
**Pull request by ojacques**
Add the DevOps Enterprise Summit, Amsterdam.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ All the events (past and coming) are available publicly in JSON: https://develop
 * 12-15: [Vue.js Live Conference](https://vuejslive.com/) - London (UK) & Online
 * 15-18: [SLOconf 2023](https://www.sloconf.com/) - Online <img alt="CFP SLOconf 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2031-January-2023&color=red">
 * 16-17: [Kafka Summit London](https://www.kafka-summit.org/events/kafka-summit-london-2023/about) - London (UK) <a href="https://sessionize.com/kafka-summit-london-2023/"><img alt="CFP Kafka Summit London 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%209-January-2023&color=red"></a>
+* 16-17: [DevOps Enterprise Summit Amsterdam](https://itrevolution.com/product/devops-enterprise-summit-amsterdam-2023/) - Amsterdam (Netherlands) <a href="[https://sessionize.com/kafka-summit-london-2023/](https://sessionize.com/devops-enterprise-summit-amsterdam-2023/)"><img alt="DevOps Enterprise Summit Amsterdam 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2013-March-2023&color=green"></a>
```
Comment (scraly) [inline README.md:159]:
```suggestion
* 16-17: [DevOps Enterprise Summit Amsterdam](https://itrevolution.com/product/devops-enterprise-summit-amsterdam-2023/) - Amsterdam (Netherlands) <a href="https://sessionize.com/kafka-summit-london-2023/"><img alt="DevOps Enterprise Summit Amsterdam 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2013-March-2023&color=green"></a>
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
Is the CFP link is correct? 😉

**Comment by ojacques** [comment]
All fixed!

**Comment by scraly** [comment]
Thanks 🙂

**Pull request merged**